src/main/java/com/epam/ms/controller/NotificationController.java

```diff
@@ -39,22 +43,29 @@
     @PostMapping
     public ResponseEntity create(@RequestBody Notification notification) {
         Notification createdNotificationId = service.create(notification);
+        String id = createdNotificationId.getId();
+        log.info("A new notification is created: /notifications/" + id);
```

Comment:
with new logging system you can say:
`log.info("A new notification is created: /notifications/{}", id);`